docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+---
+lang-ref: ch.08-1
+title: Enerjı Bazlı Modellerde Karşıtsal Yöntemler
+authors: Vishwaesh Rajiv, Wenjun Qu, Xulai Jiang, Shuya Zhao
+date: 23 Mar 2020
+traslation-date: 17 June 2020
+translator : emirceyani
+---
+<!-- title: Contrastive Methods in Energy-Based Models -->
+
+## [Tekrar](https://www.youtube.com/watch?v=ZaVP2SY23nc&t=5s)
+
+<!--## [Review](https://www.youtube.com/watch?v=ZaVP2SY23nc&t=5s) -->
+
+Dr. LeCun dersin ilk 15 dakikasında enerji bazli modelleri tekrar etmişti.
+Bu konu icin lütfen bir önceki haftayı, özellikle karşıtsal öğrenme yöntemlerini, gözden geçiriniz.
+
+<!-- Dr. LeCun spent the first ~15 min giving a review of energy-based models. Please refer back to last week (Week 7 notes) for this information, especially the concept of contrastive learning methods. -->
+
+Önceki dersten öğrendiğimiz üzere, öğrenme yöntemleri iki ana sınıfa ayrılmaktadır:
+1. Eğitim kümesindeki örneklerin enerjisini F(x_i, y_i)$ aşağı çekerken geri kalan her yerdeki enerjiyi yukarı iten $F(x_i, y’)$ Karşıtsal Yontemler.
+2. Düzenlileştirme yardımı ile sınırlı sayıda düşük enerji bölgelerine sahip bir enerji fonksiyonu F modelleyen Yapısal Yöntemler.
+
+<!-- As we have learned from the last lecture, there are two main classes of learning methods:
+1. Contrastive Methods that push down the energy of training data points, $F(x_i, y_i)$, while pushing up energy on everywhere else, $F(x_i, y’)$.
+2. Architectural Methods that build energy function F which has minimized/limited low energy regions by applying regularization. -->
+
+Farklı eğitim yöntemlerinin özelllklerini ayırt edebilmek için, Dr. Yann LeCun enerji fonksiyonunu biçimlendirmemizi sağlayan 7 strateji önermişti. Bu stratejilerden biri de veri noktalarının enerjisini aşağı çeken ve geri kalan tüm noktaların enerjisini iten En Büyük Olabilirlik(Maximum Likelihood) yöntemine benzer yöntemlerdir.
+
+<!-- To distinguish the characteristics of different training methods, Dr. Yann LeCun has further summarized 7 strategies of training from the two classes mention before. One of which is methods that are similar to Maximum Likelihood method, which push down the energy of data points and push up everywhere else. -->
+
+Maximum Likelihood yöntemi olasılıksal bir şekilde eğitim kümesindeki verilerin enerjilerini aşağı çekerken geri kalan her yerde enerjileri yukarı doğru iter. Bu yöntemö enerjilerin "mutlak değerleri"nden ziyade "aralarındaki farkı" dikkate alır. Bunun sebebi ise olasılık dağılımlarının toplamının/integralinin her zaman 1 edicek şekilde normalize edilmesidir ki bu durumda herhangi iki noktanın oranına bakmak mutlak değerlerini kıyaslamaktan daha yararlıdır.
```

Comment:
Resolved it, but I think we need to add comma to both sides of  ($y`\neq y_i$) like this: geri kalan her yerde, ($y`\neq y_i$), enerjileri . Otherwise it sounds like ($y`\neq y_i$) energies in Turkish

Reply:
added